firmware/Core/Src/littlefs/littlefs_driver.c

```diff
@@ -21,7 +21,7 @@ int LFS_block_device_read(const struct lfs_config *c, lfs_block_t block, lfs_off
 	return FLASH_read_data(
 		hspi_lfs_ptr,
 		LFS_get_chip_number(block),
-		(block * c->block_size + off),
+		((block * c->block_size) + off)/2048,
```

Comment:
Where did this 2048 magic number come from? Is it the number of bytes per block or something? Prefer to use a constant with a comment.